move/its/sources/its.move

```diff
@@ -134,11 +134,8 @@ module its::its {
     }
 
     public(package) fun coin_management_mut<T>(self: &mut ITS, token_id: TokenId): &mut CoinManagement<T> {
```

Comment:
check if there are any other borrow/mut variants that we can simplify?

Reply:
We need those for external usage.